programmers2/이상한-문자-만들기.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+    return s.split(' ').map((Index) => {
+                    let answer = '';
+                    for (let i = 0; i < Index.length; i++) {
+                         if (i % 2) answer += Index[i].toLowerCase();
+                         else answer += Index[i].toUpperCase();
+                     }
+    return answer;
+    }).join(' ');
```

Comment:
정말 잘 짜셨어요
아이디어가 너무 좋습니다.
고칠점은 줄바꿈 컨벤션,
 Index보다는 여기서는 인덱스로 사용되지 않으니 word 등이 적절해보이네요
```suggestion
    return s.split(' ').map((word) => {
        let answer = '';
        for (let i = 0; i < word.length; i++) {
          if (i % 2) answer += word[i].toLowerCase();
          else answer += word[i].toUpperCase();
        }
        return answer;
    }).join(' ');
```